safe_transaction_service/history/services/safe_service.py

```diff
@@ -45,10 +46,18 @@ class SafeCreationInfo:
     factory_address: EthereumAddress
     master_copy: Optional[EthereumAddress]
     setup_data: Optional[bytes]
+    salt_nonce: Optional[int]
     transaction_hash: str
     user_operation: Optional[aa_models.UserOperation]
 
 
+@dataclass
+class ProxyCreationData:
+    singleton: ChecksumAddress
+    initializer: bytes
+    salt_nonce: Optional[int]
```

Comment:
Should be Char as the serializer is expecting?

Reply:
Yes, being a `uint256`